core/templates/dev/head/pages/about-page/about-page.mainpage.html

```diff
@@ -1,636 +1,627 @@
-{% extends 'dist/base.html' %}
-
-{% block maintitle %}
-  About us - Oppia
-{% endblock maintitle %}
-
-{% block header_js %}
-  {{ super() }}
-{% endblock %}
-
-{% block navbar_breadcrumb %}
-  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
-    <li>
-      <span class="oppia-navbar-breadcrumb-separator"></span>
-      About
-    </li>
-  </ul>
-{% endblock navbar_breadcrumb %}
-
-{% block content %}
-  <div class="oppia-static-header">
-    <h1 translate="I18N_ABOUT_PAGE_HEADING"></h1>
-  </div>
-  <background-banner></background-banner>
-  <div ng-controller="About">
-    <div class="oppia-static-content-below-banner">
-      <div class="oppia-page-card oppia-static-content">
-        <div class="about-nav md-tab">
-          <ul class="oppia-about-tabs">
-            <li class="oppia-about-tabs-active">
-              <a ng-click="onTabClick(TAB_ID_ABOUT)" href="" class="oppia-about-tabs-text" id="about" translate="I18N_ABOUT_PAGE_TABS_ABOUT"></a>
-            </li>
-            <li>
-              <a ng-click="onTabClick(TAB_ID_FOUNDATION)" href="" class="oppia-about-tabs-text" id="foundation" translate="I18N_ABOUT_PAGE_TABS_FOUNDATION"></a>
-            </li>
-            <li>
-              <a ng-click="onTabClick(TAB_ID_CREDITS)" href="" class="oppia-about-tabs-text" id="credits" translate="I18N_ABOUT_PAGE_TABS_CREDITS"></a>
-            </li>
-          </ul>
+<!DOCTYPE html>
+<html ng-app="oppia" lang="<[currentLang]>" ng-controller="Base" itemscope itemtype="http://schema.org/Organization">
+  <head>
+    <!-- Insert head content using webpack -->
```

Comment:
I don't think this comment is necessary, but I think we should have a section in the wiki that would explain this ` @require (…)` statements in HTML files.

Reply:
Okay